tools/archetype/binding/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

```diff
@@ -1,26 +1,26 @@
-Manifest-Version: 1.0
-Bundle-ActivationPolicy: lazy
-Bundle-ClassPath: .
-Bundle-ManifestVersion: 2
-Bundle-Name: ${bindingIdCamelCase} Binding
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: ${artifactId};singleton:=true
-Bundle-Vendor: ${vendorName}
-Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
-Export-Package: 
- ${package},
- ${package}.handler
-Import-Package: 
- ${package},
- ${package}.handler,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.smarthome.config.core,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.thing,
- org.eclipse.smarthome.core.thing.binding,
- org.eclipse.smarthome.core.thing.binding.builder,
- org.eclipse.smarthome.core.thing.type,
- org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.core.util,
- org.slf4j
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: ${bindingIdCamelCase} Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: ${artifactId};singleton:=true
+Bundle-Vendor: ${vendorName}
+Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
+Export-Package:
```

Comment:
Can you readd the space at the end of this line?
Without the space the build fails with: `Error assembling JAR: invalid header field`